### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third-party application

### DISCUSSION THREAD

**Discussion by rabbularafat** [community/community#62232] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Error: "You are marked as spam, and therefore cannot authorize a third-party application"

When i attempt to authorize a third-party application (such as a desktop application) to access my GitHub account, it's encounter the error message mentioned above.

how can i fixed it?

**Comment by cehfisher** [comment]
Hi @rabbularafat - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
